app/src/main/java/com/pwr266521/bmiapp/BMIActivity.kt

```diff
@@ -0,0 +1,32 @@
+package com.pwr266521.bmiapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import com.pwr266521.bmiapp.util.BMIUtil.BMI_KEY
+import com.pwr266521.bmiapp.util.BMIUtil.bmiDescriptions
+import com.pwr266521.bmiapp.util.BMIUtil.getBMICategory
+
+class BMIActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_bmiactivity)
+
+        val goBackBtn: Button = findViewById(R.id.go_back_button)
```

Comment:
Gdybyś wykorzystał fragmenty i [Navigation component](https://developer.android.com/guide/navigation/get-started) to w dość prosty sposób móglbyś mieć wspolny top bar i automatycznie pojawiłby się na nim back button